mapie/classification.py

```diff
@@ -500,6 +523,138 @@ def _add_random_tie_breaking(
         )
         return prediction_sets
 
+    def _fix_number_of_classes(
+        self,
+        est_classes: ArrayLike,
+        y_proba: ArrayLike
+    ) -> ArrayLike:
+        """
+        Fix shape of y_proba of validation set if number of classes
+        of the training set used for cross-validation is different than
+        number of classes of the original dataset y.
+
+        Parameters
+        ----------
+        est_classes : ArrayLike
+            Classes of the training set.
+        y_val_proba : ArrayLike
+            Probabilities of the validation set.
+
+        Returns
+        -------
+        ArrayLike
+            Probabilities with the right number of classes.
+        """
+        y_proba_tot = np.zeros(
```

Comment:
`y_pred_tot` is misleading, because it suggest that there is a summation somewhere. Maybe `y_pred_full` is more explicit ? Or `y_pred_padded` ?

Reply:
Let's go for `y_pred_full`.